src/renderer/components/+custom-resources/crd.store.ts

```diff
@@ -1,23 +1,29 @@
 import { computed, reaction } from "mobx";
 import { KubeObjectStore } from "../../kube-object.store";
 import { autobind } from "../../utils";
-import { crdApi, CustomResourceDefinition } from "../../api/endpoints/crd.api";
+import { crdBetaApi, CustomResourceDefinition } from "../../api/endpoints/crd.api";
 import { apiManager } from "../../api/api-manager";
 import { KubeApi } from "../../api/kube-api";
 import { CRDResourceStore } from "./crd-resource.store";
 import { KubeObject } from "../../api/kube-object";
 
+function initStore(crd: CustomResourceDefinition) {
+  const apiBase = crd.getResourceApiBase();
+  const [kind, isNamespaced] = [crd.getResourceKind(), crd.isNamespaced()];
+  const api = apiManager.getApi(apiBase) || new KubeApi({ apiBase, kind, isNamespaced });
+  const store = apiManager.getStore(api) || new CRDResourceStore(api);
+  apiManager.registerStore(api, store);
+}
+
 @autobind()
 export class CRDStore extends KubeObjectStore<CustomResourceDefinition> {
-  api = crdApi
+  api = crdBetaApi
```

Comment:
Why do we use `crdBetaApi` here?

Reply:
Because I renamed `crdApi` to `credBetaApi` before adding the non-beta version.

Reply:
okay...should we use `crdApi` and have `crdBetaApi` in the background. Feels more natural that way.

Reply:
Okay, I did this so that `CRDStore` didn't change versions, but changing makes more sense.